webassembly/app.html

```diff
@@ -223,6 +221,51 @@ <h1 class="title">F3D Web</h1>
         // do a first render and start the interactor
         Module.engineInstance.getWindow().render();
         Module.engineInstance.getInteractor().start();
+
+        // Parse mesh from url-param search query hash
+        function filename_for_model_url(model_url, extension_parsed, contentDisposition) {
+          // Build filename given extension urlparam or response header content-disposition
+          if (extension_parsed) {
+            return `model_urlparam.${extension_parsed}`;
+          } else {
+            // If extension is not provided by user, try to get it auto from content-disposition header of url extension 
+            if (contentDisposition) {
+              return contentDisposition.split('filename=')[1].split(';')[0];
+            } else {
+              return model_url.split('/').pop();
+            }
+          }
+          return filename;
+        }
+        function load_from_url(){
+          // Parse search-query mesh url-param or load default mesh file
+          // const params = new URLSearchParams(window.location.search);
+          // Replace first hash with question mark to have real search query parsing and avoid leading # in first parsed urlparam
+          const params = new URLSearchParams(window.location.hash.replace(/^#/, '?'));
+          const model_url_passed = params.get("mesh");
```

Comment:
files are not only meshes, they can also contain materials and lights.  
Can we rename the option to `model` instead?